ajax/libs/webicons/package.json

```diff
@@ -11,5 +11,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/adamfairhead/webicons.git"
-  }
+  },
+  "license": "CC-BY-3.0-US"
```

Comment:
I only find `CC-BY-3.0` in following license list
https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L63
https://spdx.org/licenses/CC-BY-3.0.html

so I need to separate the license to `CC-BY-3.0` and US part?